scoop_protocol/scoop_protocol.go

```diff
@@ -43,6 +43,11 @@ type LoadCheckResponse struct {
 	ManifestURL string
 }
 
+//Scoop healthcheck for ingester healthcheck json unmarshal
+type ConnError struct {
```

Comment:
Hmm, ConnError doesn't tell me very much about this. I think this is ScoopHealthCheck